### DISCUSSION TITLE
Home Assistant OS 12.0.rc1

### DISCUSSION THREAD

**Discussion by sairon** [home-assistant/operating-system#3178] [Announcements]
## Home Assistant Operating System

* Use verity bundle format for RAUC updates (#3152) @sairon
* Bump buildroot to update BlueZ to v5.72 (#3147) @sairon

## Build System

* Move output directory to /mnt in GH build action (#3174) @sairon
* Bump Pi 5 image on release in Raspberry Pi Imager json (#3161) @agners
* Bump release-drafter/release-drafter from 5 to 6 (#3144) @dependabot

## Raspberry Pi

* Add udev rule to rpi-rf-mod package  (#2854) @jens-maus
* RaspberryPi: Update kernel to 6.1.73 - stable_20240124 (#3162) @sairon

## Home Assistant Yellow

* RaspberryPi: Update kernel to 6.1.73 - stable_20240124 (#3162) @sairon

## Home Assistant Green

* Factor out common Rockchip config (#3165) @darkxst
* Linux: Update kernel to 6.6.16 (#3159) @sairon
* Enable ath12k, MT7996E and various new Realtek drivers from Linux v6.6 (#3151) @sairon

## Open Virtual Appliance

* Linux: Update kernel to 6.6.16 (#3159) @sairon
* Enable ath12k, MT7996E and various new Realtek drivers from Linux v6.6 (#3151) @sairon

## Generic x86-64

* Add SD/MMC drivers of Realtek card readers for generic_x86_64 (#3170) @sairon
* Linux: Update kernel to 6.6.16 (#3159) @sairon
* Enable ath12k, MT7996E and various new Realtek drivers from Linux v6.6 (#3151) @sairon

## Hardkernel ODROID

* Add support for Odroid M1S Board (#3121) @darkxst
* Factor out common Rockchip config (#3165) @darkxst
* Linux: Update kernel to 6.6.16 (#3159) @sairon
* Fix ODROID XU4 boot by disabling PROVE_LOCKING in kernel (#3158) @sairon
* Enable ath12k, MT7996E and various new Realtek drivers from Linux v6.6 (#3151) @sairon

## ASUS Tinker

* Linux: Update kernel to 6.6.16 (#3159) @sairon
* Enable ath12k, MT7996E and various new Realtek drivers from Linux v6.6 (#3151) @sairon

## Khadas VIM Series

* Linux: Update kernel to 6.6.16 (#3159) @sairon
* Enable ath12k, MT7996E and various new Realtek drivers from Linux v6.6 (#3151) @sairon

## Generic aarch64

* Linux: Update kernel to 6.6.16 (#3159) @sairon
* Enable ath12k, MT7996E and various new Realtek drivers from Linux v6.6 (#3151) @sairon

## Other Changes

* Update issue template with better links to logs, add CLI instructions (#3169) @sairon
* Use separate path for v6.1.y and v6.6.y kernel config fragments (#3146) @sairon

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/12.0.rc1'>Home Assistant OS 12.0.rc1</a>.</em>

**Comment by Precicion1** [comment]
Got it working on Raspberry pi 5 while I am a noob. Whoehoeeeeeee 👯‍♀️

**Comment by angainor** [comment]
I just updated to 12.0.rc1. Changelog says kernel is updated to 6.6, but I see this:
```
# uname -a
Linux homeassistant 6.1.73-haos-raspi #1 SMP PREEMPT Thu Feb 15 12:03:07 UTC 2024 aarch64 HAOS

# ha info
arch: aarch64
channel: stable
docker: 24.0.7
features:
- reboot
- shutdown
- services
- network
- hostname
- timedate
- os_agent
- haos
- resolved
- journal
- disk
- mount
hassos: 12.0.rc1
homeassistant: 2024.2.2
hostname: homeassistant
logging: info
machine: raspberrypi4-64
operating_system: Home Assistant OS 12.0.rc1
state: startup
supervisor: 2024.01.1
supported: true
supported_arch:
- aarch64
- armv7
- armhf
timezone: Europe/Oslo
```
So is the kernel really upgraded, and if so - why does my HA think it's not?

**Reply by sairon** [reply]
See the Raspberry Pi section - its kernel was only updated to 6.1.73, as it's using [Raspberry Pi stable downstream kernel](https://github.com/raspberrypi/linux/).

**Reply by angainor** [reply]
I see. FYI, this did seen to solve the Bluetooth connection issue for me. So far so good.

**Comment by bopong** [comment]
Any idea how long the HAOS_yellow-12.0.rc1.img takes to install on a RaspberryPi.  Mine has been flashing yellow in the heartbeat pattern for almost 2 hours installing.  Is that normal?

**Reply by agners** [reply]
Updating usually only takes 1-2 minutes, plus the time it takes to reboot the system. For a new installation things take a bit longer, but not really more than 5 minutes I'd say.

Are you running an update or a new installation? Is the frontend stuck or the device really no longer reachable?

**Reply by bopong** [reply]
Its up and running normally but suprisingly I never had to disconnect and replug it in.  I think what threw me off was it did the normal light sequence but now has a steady heartbeat of the yellow lights while running which it didn't have before.   Before  I had to reset it was just the solid red LED when it was running.

**Comment by cyclobob** [comment]
I just cannot get this to run. First installed on an SD card using Balena, but the Rpi5 just hangs on start. It stays with the Ethernet light on (yellow one) and the system LED on greeen. Left it for 12 hours no joy. Tried using an SSD I flashed from my mac using the pi imager and 11.5 preview. Exact same issue. If I run the standard Pi OS everything is fine. Any suggestions? Could I have a rogue pi5?

**Comment by ClintSiren** [comment]
I just upgraded form 11.4Dev to 12RC1 and all seems to have gone well. Im running the Pi5. 
I ran a restart with no issues as well as a hard reboot all systems are up and running, i have been scared of re installing so have been holding off since my last full reinstall.

Thanks guys, for all the effort!

Core
2024.2.3
Supervisor
2024.02.0
Operating System
12.0.rc1
Frontend
20240207.1
